README.md

```diff
@@ -11,7 +11,7 @@ package.
 The easiest way to test it out is to pick up the latest
 Fedora CoreOS preview artifact from:
 
-http://artifacts.ci.centos.org/fedora-coreos/prod/builds/latest/
+https://builds.coreos.fedoraproject.org/browser
```

Comment:
Let's link to https://getfedora.org/coreos/download/ instead of the development build browser.